server/app.js

```diff
@@ -21,47 +25,71 @@ const server = http.createServer(app);
 
 const io = socketio(server, {
   cors: {
-    origin: "*",
+    origin: '*',
   },
 });
 
-io.on("connection", (socket) => {
-  console.log("connected");
+io.on('connection', (socket) => {
+  console.log('connected');
 });
 
-if (process.env.NODE_ENV === "development") {
-  app.use(logger("dev"));
+if (process.env.NODE_ENV === 'development') {
+  app.use(logger('dev'));
 }
 app.use(json());
 app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(join(__dirname, "public")));
+app.use(express.static(join(__dirname, 'public')));
 
 app.use((req, res, next) => {
   req.io = io;
   next();
 });
 
-app.use("/auth", authRouter);
-app.use("/users", userRouter);
+app.use('/upload-images', upload.array('image'), async (req, res) => {
+  const uploader = async (path) => await cloudinary.uploads(path, 'Images');
+
+  if (req.method === 'POST') {
+    const urls = [];
+    const files = req.files;
+    for (const file of files) {
+      const { path } = file;
+      const newPath = await uploader(path);
+      urls.push(newPath);
+      fs.unlinkSync(path);
+    }
+
+    res.status(200).json({
+      message: 'images uploaded successfully',
+      data: urls,
+    });
+  } else {
+    res.status(405).json({
+      err: `${req.method} method not allowed`,
+    });
+  }
+});
```

Comment:
You can use a post method ( `app.post`) here so you don't need to be checking the type of request for returning a response.